kubernetes/samples/charts/util/setupLoadBalancer.sh

```diff
@@ -228,7 +258,9 @@ function createNginx() {
   count=0
   tpod=$(kubectl get po -n ${NSPACE} --no-headers | awk '{print $1}')
   while test $count -lt $max; do
-    if test "$(kubectl get po -n ${NSPACE} --no-headers | awk '{print $2}')" = 1/1; then
+    status=$(kubectl get po -n ${NSPACE} --no-headers 2> /dev/null \
+                | awk '{print $2}')
+    if [ ${status} == "1/1" ]; then
       echo "Nginx operator pod is running now."
       kubectl get pod/${tpod} -n ${NSPACE}
       kubectl exec -it $tpod -n nginx -- /nginx-ingress-controller --version``
```

Comment:
change this to:
kubectl exec -it $tpod -n ${NSPACE} -- /nginx-ingress-controller --version``

Reply:
done